File: php/Processor.php
<?php

class Processor{
    
    // deklarasi dan inisialisasi atribut private
    private $name = "-";
    private $price = 0;

    // constructor dengan parameter
    public function __construct($inputName = "", $inputPrice = 0){
        $this->name = $inputName;
        $this->price = $inputPrice;
    }


    // deklarasi setter sebagai method (Write)
    public function setName($name){
        $this->name = $name;
    }
    public function setPriceP($price){
        $this->price = $price;
    }

    
    // deklarasi getter sebagai method (Read Only)
    public function getName(){
        return $this->name;
    }
    public function getPriceP(){
        return $this->price;
    }

    // deklarasi method Display untuk menampilkan data menggunakan method Getter
    public function displayProcessor(){
        echo "Processor      = ". $this->getName(). "<br>";
    }

    
    // destructor
    public function __destruct(){
        
    }
}

?>

File: python/Processor.py

class Processor():

    # deklarasi dan inisialisasi atribut private
    __name = []
    __price = 0


    # constructor tanpa parameter
    def __init__(self):
        __name = []
        __price = 0

    # constructor dengan parameter
    def __init__(self, name = [], price = 0):
        
        self.__name = name
        self.__price = price


    # deklarasi setter sebagai method (Write)
    def setName(self, name):
        self.__name = name

    def setPrice(self, price):
        self.__price = price

    
    # deklarasi getter sebagai method (Read Only)
    def getName(self):
        return self.__name

    def getPriceP(self):
        return self.__price
    

    # deklarasi method Display untuk menampilkan data menggunakan method Getter
    def displayProcessor(self):
        print("Processor      = " + str(self.getName()))